models/ValidationManager.cfc

```diff
@@ -50,8 +50,6 @@
  * </pre>
  *
  */
-import cbvalidation.models.*;
```

Comment:
I just thought we don't need it, because we are using wirebox to instantiate objects where we don't need the full path.
But I see there are references to ValidationResult and ValidationError in the code, so maybe just add them back.
And what about all Interfaces stuff like IvalidationError and IvalidationResult? Should we just keep them as return values? In that case, shouldn't we add the interfaces to the imports as well?

To be honest, I didn't do a lot with imports until know, so please make me smarter :-)

Reply:
😄  JAJAJAJA you are too funny @wpdebruin ! 

So as far as the interfaces to, we remove them from the return type and move the return to `any` then in the cfc docs we document it, example:

```
IValidationResult function test()

Now becomes

/**
 * Functon here
 *
 * @return cbvalidation.interfaces.IValidationResult
 */
any function test()

```

Now, as far the imports go, they allow you to use shorthand notations for classes.  Example:, we import `cbvalidation.models.result.*;` This allows me in CODE to use the CFC located within the `result` package by name instead of full path since you told the compiler to import the CFC paths into your CFC blueprint.

```
// With Import
IValidationResult function getResults()

// Without Import
cbvalidation.models.result.IValidationResult function getResult()
```

Does that make sense?

Reply:
Yes, all clear. The whole idea about the imports and shorthand notation is exactly what I thought, but I was wondering if I missed something.
I think it is indeed a good idea to move the return type to any and just document the interfaces. That looks a lot more familiar if I compare it to more recent modules. There's still quite some legacy stuff in cbvalidation but I guess that will change when we take the next steps in rebuilding.
All clear for now, I 'll do some updates and then continue with the next part (the cbi18n stuff)

Reply:
moved interfaces and brought back the imports. Ready for the next round.